apps/frontend/src/app/rooms/room-list/room-list.component.html

```diff
@@ -1,131 +1,165 @@
         <mat-select-trigger>
           {{
-            isDesktop
-              ? ((filterFaculty.value ? 'FACULTY.' : '') +
-                  filterFaculty.value || '' | translate)
-              : filterFaculty.value
+          isDesktop
+            ? ((filterFaculty.value ? 'FACULTY.' : '') +
+            filterFaculty.value || '' | translate)
+            : filterFaculty.value
           }}
         </mat-select-trigger>
-        <mat-option *ngFor="let faculty of facultyList" [value]="faculty">{{
+        <mat-option *ngFor='let faculty of facultyList' [value]='faculty'>{{
           isDesktop
             ? ((faculty ? 'FACULTY.' : '') + faculty || '' | translate)
             : faculty
-        }}</mat-option>
+          }}</mat-option>
       </mat-select>
     </mat-form-field>
-
+  </div>
+  <div class='filter-actions'>
     <button
       mat-stroked-button
-      color="warn"
-      (click)="resetFilter()"
-      class="filter-space"
+      color='warn'
+      (click)='resetFilter()'
+      class='filter-space'
     >
       <mat-icon>replay</mat-icon>
     </button>
-  </div>
-
-  <div class="filter-actions">
     <button
       mat-raised-button
-      *ngIf="adminSrv.getIsAdmin()"
-      (click)="openFormDialog($event)"
+      *ngIf='adminSrv.getIsAdmin()'
+      (click)='openFormDialog($event)'
     >
       <mat-icon>add</mat-icon>
     </button>
   </div>
+
+
 </div>
-<table mat-table [dataSource]="roomList" class="mat-elevation-z8">
-  <ng-container matColumnDef="name">
-    <mat-header-cell *matHeaderCellDef>Name</mat-header-cell>
-    <mat-cell *matCellDef="let room">{{ room.name }}</mat-cell>
+
+<table mat-table [dataSource]='roomList' class='mat-elevation-z8'>
+  <ng-container
+    matColumnDef='name'>
+    <mat-header-cell
+      [ngClass]="{ 'padding-left-12': !isDesktop }"
+      *matHeaderCellDef>Name
+    </mat-header-cell>
+    <mat-cell
+      [ngClass]="{ 'padding-left-12': !isDesktop }"
+      *matCellDef='let room'>{{ room.name }}</mat-cell>
   </ng-container>
 
-  <ng-container matColumnDef="updated">
+  <ng-container matColumnDef='updated'
+                *ngIf='isDesktop'>
     <mat-header-cell *matHeaderCellDef>{{
       'ROOMS.UPDATE_ROOM' | translate
-    }}</mat-header-cell>
-    <mat-cell *matCellDef="let room">{{
+      }}</mat-header-cell>
+    <mat-cell *matCellDef='let room'>{{
       room.updatedAt | customDate
-    }}</mat-cell>
+      }}</mat-cell>
   </ng-container>
 
-  <ng-container matColumnDef="maxParticipants">
-    <mat-header-cell *matHeaderCellDef>{{
+  <ng-container matColumnDef='maxParticipants'>
+    <mat-header-cell
+      [ngClass]="{ 'center-text': !isDesktop }"
+      *matHeaderCellDef>{{
       'ROOMS.MAX_PARTICIPANTS' | translate
-    }}</mat-header-cell>
-    <mat-cell *matCellDef="let room">{{ room.maxParticipants }}</mat-cell>
+      }}</mat-header-cell>
+    <mat-cell
+      [ngClass]="{ 'center': !isDesktop }"
```

Comment:
Damit der Abstand auf mobile auch passt @bilal0710 
```suggestion
      [ngClass]="{ 'center-text': !isDesktop }"
```